presto-hive-common/src/main/java/com/facebook/presto/hive/HiveCommonClientConfig.java

```diff
@@ -15,10 +15,238 @@
 
 import com.facebook.airlift.configuration.Config;
 import com.facebook.airlift.configuration.ConfigDescription;
+import com.facebook.presto.orc.OrcWriteValidation.OrcWriteValidationMode;
+import com.facebook.presto.spi.schedule.NodeSelectionStrategy;
+import io.airlift.units.DataSize;
+
+import javax.validation.constraints.DecimalMax;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
+
+import static com.facebook.presto.spi.schedule.NodeSelectionStrategy.NO_PREFERENCE;
+import static io.airlift.units.DataSize.Unit.MEGABYTE;
 
 public class HiveCommonClientConfig
 {
+    private boolean isReadNullMaskedParquetEncryptedValueEnabled;
```

Comment:
Rename it as readNullMaskedParquetEncryptedValueEnabled and move it to be after rangeFiltersOnSubscriptsEnabled